proficio_sandbox/proficio.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python
```

Comment:
We should consider deleting this, now that we have a more full-featured Proficio launcher.